### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### LINKED ISSUE #870: Need to improve PR template
Working on it

### PR DESCRIPTION
**Pull request by saarthakchats**
Improved PR template

Fixes #870

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
+- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
+- Fixes`#<Issue no.>`
 
-<!-- Changes: Add here what changes were made in this issue and if possible provide links. -->
--
 
-<!-- Demo Link: Add here the link where you changes can be seen. -->
--
+# Demo Link: Add here the link where you changes can be seen.
+  [Link Name](url)
```
Comment (harshit98) [inline .github/PULL_REQUEST_TEMPLATE.md]: Remove `[Link Name](url)`

**Comment by saarthakchats** [comment]
@harshit98

**Comment by saarthakchats** [comment]
@nikhilrayaprolu 
Pls merge

**Comment by saarthakchats** [comment]
Can you tell me the line, or what is causing the failure, I dont know how to find

**Comment by saarthakchats** [comment]
@isuruAb

**Comment by saarthakchats** [comment]
Pls merge this @isuruAb

**Pull request closed without merging**